srcpkgs/dracut/files/kernel-hook-postinst

```diff
@@ -7,9 +7,9 @@
 PKGNAME="$1"
 VERSION="$2"
 
-if [ ! -x bin/dracut ]; then
+if [ ! -x usr/bin/dracut ]; then
 	exit 0
 fi
```

Comment:
I don't like this, because the hook should only be there if dracut itself is installed. @sgn and I didn't include it in the `dracut-uefi` hook.